docs/deep-dives/github.md

```diff
@@ -43,18 +43,12 @@ steps:
 ```
 
 With Docker Compose,
-declare the environment variables and volume mounts in your [Compose file]:
+declare volume mounts in your [Compose file].
+You won't need to declare the environment variables in the compose file anymore since `propagate-environment: true` will ensure the variables defined are propagated to the docker environment.
```

Comment:
Can we document how a step with the `docker-compose` plugin should look like, similar to the example above for the `docker` plugin?